admin/src/js/main.js

```diff
@@ -168,6 +168,37 @@ angular.module('adminApp').config(function(
   $ngReduxProvider.createStoreWith(RootReducer, []);
 
   $stateProvider
+    .state('display', {
+      url: '/display',
+      templateUrl: 'templates/display.html'
+    })
+    .state('display.date-time', {
+      url: '/date-time',
+      views: {
+        tab: {
+          controller: 'SettingsBasicCtrl',
+          templateUrl: 'templates/settings_basic.html'
```

Comment:
This is meant to show the date settings from the "Settings > Advanced" tab, not the "Settings > Basic" tab.